consensus/polybft/polybft.go

```diff
@@ -211,6 +213,7 @@ func (p *Polybft) Initialize() error {
 
 // Start starts the consensus and servers
 func (p *Polybft) Start() error {
+	fmt.Println(DisclaimerMessage)
```

Comment:
Had same doubt to be honest, but if you have it too, then probably the logger it is :slightly_smiling_face:

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1107/commits/d09c14a53978821b32e51817a3d429e276620d76
Since genesis cmd doesn't have a logger instance, disclaimer is printed as part of CLI output.